lib/chef/resource/windows_security_policy.rb

```diff
@@ -83,6 +84,59 @@ class WindowsSecurityPolicy < Chef::Resource
       description: "Policy value to be set for policy name."
 
       load_current_value do |desired|
+        current_state = load_security_options
+
+        if desired.secoption == "ResetLockoutCount"
+          if desired.secvalue.to_i > 30
```

Comment:
You might want to just do this with a callback right in the property:

https://docs.chef.io/custom_resources/#validators